README.md

```diff
@@ -1,22 +1,126 @@
+
+Launchpad can build any image, publish it to your Docker registry, and deploy it to Kubernetes in one step.
+
+### A consistent deployment setup per project
+
+Launchpad defines your project's Kubernetes configuration. Whether you are deploying in development, staging, or production environment, whether you are deploying to a local or remote Kubernetes cluster, Launchpad always creates the same set of Kubernetes resources for you each time.
+
+### Easy onboarding for everyone on the team
+
+Launchpad gives each developer on your team a dedicated namespace to work on and deploy to and generates unique preview URLs for easy testing.
+
+### Secret management built-in
+
+Secrets are tied to your launchpad projects, so they can be shared and updated securely by your team.
+
+## Installing Launchpad
+
+In addition to installing Launchpad itself, you will need to install `docker` since Launchpad depends on it:
+
+1. Install [Docker Desktop](https://www.docker.com/get-started/).
+
+2. Install Launchpad:
+
+   ```sh
+   curl -fsSL https://get.jetpack.io/launchpad | bash
+   ```
+
+Read more on the [Launchpad docs](http://www.jetpack.io/launchpad/docs/getting-started/any-image-to-k8s-5-min/).
+
+
+## Quickstart: deploy to your Docker Desktop Kubernetes cluster
+
+In this quickstart, we’ll deploy a cron job to your local Docker Desktop Kubernetes cluster.
+
+1. Open a terminal in a new empty folder called `app/`.
+
+2. Enable [Kubernetes on Docker Desktop](https://docs.docker.com/desktop/kubernetes/)
+
+3. Initialize Launchpad in `app/`:
+
+   ```bash
+   > launchpad init
+   ```
+   
+   You will see the following questions:
+   ```
+   ? What is the name of this project? // Press <Enter> to use the default name
+   ? What type of service you would like to add to this project? // Choose `Cron Job`
+   ? To which cluster do you want to deploy this project? // Choose `docker-desktop`
+   ```
+   
+   
+   This creates a `launchpad.yaml` file in the current directory. You should commit it to source control.
+
+4. Your `launchpad.yaml` file should now look like this:
+
+   ```yaml
+      configVersion: 0.1.2
+      projectId: ...
+      name: app
+      cluster: docker-desktop
+      services:
+        app-cron:
+          type: cron
+          image: busybox:latest
+          command: [/bin/sh, -c, date; echo Hello from Launchpad]
+          schedule: '* * * * *'
+   ```
+
+5. Start a new deployment to Kubernetes:
+
+   ```bash
+   launchpad up
+   ```
+
+6. Wait for a minute, and see the cron job in action:
+
+   ```bash
+   > kubectl get pods
+   > kubectl logs <pod_name>
+   ```
+
+   [kubectl](https://www.jetpack.io/launchpad/docs/getting-started/any-image-to-k8s-5-min/#prerequisites), while not required, is a useful tool for inspecting and managing your deployments in Kubernetes.
+
+
+## Additional commands
+
+`launchpad help` - see all commands
+
+`launchpad env` - use launchpad's secret management toolchain
```

Comment:
Should we flag some as Mission Control required?

Reply:
Sure. I'm thinking people don't really know what Mission Control really is, so that information may not be helpful. I can say, `login` required maybe?

Reply:
Yeah, let's do that instead. Otherwise LGTM